cloud/jenkins/pgo_operator_gke_version.groovy

```diff
@@ -219,17 +219,8 @@ void runTest(Integer TEST_ID) {
                         export IMAGE_PGBADGER=${PGO_PGBADGER_IMAGE}
                     fi
 
-                    if [ -n "${PMM_SERVER_IMAGE_BASE}" ]; then
-                        export IMAGE_PMM_SERVER_REPO=${PMM_SERVER_IMAGE_BASE}
-                    fi
-
-                    if [ -n "${PMM_SERVER_IMAGE_TAG}" ]; then
-                        export IMAGE_PMM_SERVER_TAG=${PMM_SERVER_IMAGE_TAG}
-                    fi
-
-                    if [ -n "${PMM_CLIENT_IMAGE}" ]; then
-                        export IMAGE_PMM=${PMM_CLIENT_IMAGE}
-                    fi
+                    export IMAGE_PMM_CLIENT=$IMAGE_PMM_CLIENT
```

Comment:
Why this one has different approach then the above ones?

Reply:
done